zilliqa-macros/src/test.rs

```diff
@@ -17,13 +17,18 @@ pub(crate) fn test_macro(_args: TokenStream, item: TokenStream) -> TokenStream {
     input.sig.ident = inner_name.clone();
 
     quote! {
+        use ntest::*;
```

Comment:
Use a * here to avoid complaints about re-importing multiple times. Now, tests have a timeout before they fail with a message about the test taking too long.